src/Servers/IIS/IIS/src/Core/IISHttpContext.IO.cs

```diff
@@ -135,6 +135,12 @@ private async Task ReadBody()
             AbortIO(clientDisconnect: true);
             error = ex;
         }
+        catch (Http.BadHttpRequestException ex)
+        {
+            // Similar to a ConnectionResetException, this shouldn't be logged as an "Unexpected exception."
+            // This should be logged by whatever catches it. Likely IISHttpContextOfT.ProcessRequestsAsync().
+            error = ex;
```

Comment:
Are there existing unit tests that check `BadHttpRequestException` is logged? The unit tests in the PR all seem to test ApplicationError

Reply:
The tests that I updated in are logging MaxRequestBodySizeTests the `BadHttpRequestException`'s as ApplicationErrors. I'll update the test to verify the exception type.